### ISSUE TITLE
Accessing non-existent property 'MongoError' of module exports inside circular dependency

### ISSUE THREAD

**Issue by QuentinGruber** [QuentinGruber/h1z1-server#101]
**Describe the bug**
A warning show up.
 ```
(node:10788) Warning: Accessing non-existent property 'MongoError' of module exports inside circular dependency
(Use `node --trace-warnings ...` to show where the warning was created)
 ```

**To Reproduce**
Launch Solo Server from H1emu-server-app with nodejs 14.15

**Expected behavior**

**Screenshots**
![image](https://user-images.githubusercontent.com/47059878/106812437-f56e9480-666f-11eb-847b-2399787565e8.png)

**Comment by QuentinGruber** [comment]
seems like it's an issue from mongodb ( only a warning tho )

**Comment by QuentinGruber** [comment]
fixed on Mongodb 3.6.5 🎉  #132
